srcpkgs/kdiff3/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'kdiff3'
+pkgname=kdiff3
+version=0.9.98
+revision=1
+build_style=gnu-configure
+#build_wrksrc=dist
+makedepends="qt-qmake qt-devel qt-devel-tools"
+short_desc="Both kdiff and kdiff2 name taken"
+maintainer="Juan RP <xtraeme@voidlinux.eu>"
+license="GPL-2"
+homepage="http://kdiff3.sourceforge.net/"
+distfiles=http://sourceforge.net/projects/kdiff3/files/kdiff3/${version}/kdiff3-${version}.tar.gz
+checksum=802c1ababa02b403a5dca15955c01592997116a24909745016931537210fd668
+
+do_configure() {
+	echo "Not a real configure, step ignored, but must keep this"
+}
+do_build() {
+	export OPTFLAG="$CFLAGS"
+	./configure qt4
```

Comment:
have this in do_configure

Reply:
Can't. Because this project used a fake configure - it's just a shell script masking a qmake project.